src/renderer/pages/Staking/Operations/components/Modals/SelectedValidatorsModal/SelectedValidatorsModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { Accordion, BaseModal } from '@renderer/shared/ui';
+import { useI18n } from '@renderer/app/providers';
+import { Validator } from '@renderer/shared/core/types/validator';
+import type { Explorer } from '@renderer/shared/core';
+import { ValidatorsTable } from '@renderer/entities/staking/ui';
+
+type Props = {
+  isOpen: boolean;
+  validators: Validator[];
+  explorers?: Explorer[];
+  onClose: () => void;
+};
+
+export const SelectedValidatorsModal = ({ isOpen, validators, explorers, onClose }: Props) => {
+  const { t } = useI18n();
+
+  return (
+    <BaseModal
+      closeButton
+      contentClass="pb-3 pt-2"
+      panelClass="w-[368px] max-h-[660px] overflow-x-hidden"
+      title={t('staking.confirmation.validatorsTitle')}
+      isOpen={isOpen}
+      onClose={onClose}
+    >
+      <div className="flex flex-col gap-y-2 mt-2">
```

Comment:
These styles have an effect?
`div` has a single element inside
If no, `mt-2` could be moved to `<Accordion className="mt-2">`

Reply:
I'll delete this `div` it's not needed here, thanks!